spec/support/square.rb

```diff
@@ -29,6 +29,9 @@ def order_payload(order)
     {
       idempotency_key: SecureRandom.uuid,
       order: {
+        metadata: {
+          hosted_checkout: "true"
+        },
```

Comment:
Instead of having it static, I think it would be better to let to set it from the params:

```
def find_or_create_square_order_id_on_sandbox(order, hosted_checkout: false)
....
  order_payload = order_payload(order)
  order_payload[:metadata] = { hosted_checkout: 'true' } if hosted_checkout

  client.orders.create_order(body: order_payload).data.order[:id]
```

Reply:
https://github.com/nebulab/solidus_square/pull/43/commits/c7a1796d2cb07146aa601d9fb3e79b4de1a5f9c8